inc/nux/class-storefront-nux-starter-content.php

```diff
@@ -354,14 +354,14 @@ public function wc_query( $query ) {
 			$post__in = array();
 
 			// Add existing products.
-			$existing_products = $this->_get_existing_wc_products();
+			$existing_products = $this->get_existing_wc_products();
 
 			if ( ! empty( $existing_products ) ) {
 				$post__in = array_merge( $post__in, $existing_products );
 			}
 
 			// Add starter content.
-			$created_products = $this->_get_created_starter_content_products();
+			$created_products = $this->get_createdstarter_content_products();
```

Comment:
I see there's still a little typo / search-replace issue here – this should be `get_created_starter_content_products()`, right?

Reply:
Yes, `get_createdstarter_content_products()` is definitely wrong. Not sure what happened in this case, but somehow, I broke it.